### PR TITLE
docs(MAINTAINING): add instructions for becoming a maintainer

### PR DESCRIPTION
**Pull request by zetok**
Also add steps for reviewing PRs and slightly adjust formatting.

cc:
- maintainers:  @antis81 @sudden6 @tux3
- people interested in becoming maintainers: @BiTOk @Diadlo @initramfs

---

This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/3599)

### CODE REVIEW

File: MAINTAINING.md [hunk 1]
```diff
@@ -97,6 +118,19 @@ git config --global alias.logs 'log --show-signature'
   duplicate of, tag with higher `duplicates:#`
 
 
+# How to become a maintainer?
+
+Contribute, review & test pull requests, be active, oh and don't forget to
+mention that you would want to become a maintainer :P
+
+Aside from contents of [`CONTRIBUTING.md`] you should also know content of this
```
Comment (tux3) [inline MAINTAINING.md]: Nitpick: s/know content/know the contents/

Comment (zetok) [inline MAINTAINING.md]: Fixed.

**Comment by tux3** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3599#-:-KOWN1kkPN6SDAz8aCbp:bnfp4nl)_

**Comment by tux3** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3599)_

**Comment by zetok** [comment]
Note: @initramfs & @Diadlo have been invited to qTox org.

**Pull request merged**